features/networking/ovn_ipsec.feature

```diff
@@ -157,3 +157,68 @@ Feature: OVNKubernetes IPsec related networking scenarios
+    Given I have a project
+    Given I obtain test data file "networking/pod-for-ping.json"
+    When I run oc create over "pod-for-ping.json" replacing paths:
+      | ["spec"]["nodeName"] | <%= cb.workers[1].name %> |
+      | ["metadata"]["name"] | pod-worker1               |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+       | name=hello-pod |
+    And evaluation of `pod.ip_url` is stored in the :test_pod_worker1 clipboard
+    
+    Given I obtain test data file "networking/pod-for-ping.json"
+    When I run oc create over "pod-for-ping.json" replacing paths:
+      | ["spec"]["nodeName"]                 | <%= cb.workers[0].name %>                                                                     |
+      | ["metadata"]["name"]                 | pod-worker0                                                                                   |
+      | ["spec"]["containers"][0]["command"] | ["bash", "-c", "for f in {0..3600}; do curl <%= cb.test_pod_worker1 %>:8080 ; sleep 1; done"] |
+    Then the step should succeed
+    #Above command will curl "hello openshift" traffic every 1 second to worker1 test pod which is expected to cause ESP traffic generation across those nodes
+    And a pod becomes ready with labels:
+       | name=hello-pod |
+
+    #Enable ipsec through CNO
+    Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with:
+      | {"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipsecConfig":{}}}}} |  
+    #Make sure you are receiving ESP packets at the destination node. For that we will simulate a prviledged pod to allow tcpdumping
+    Given I obtain test data file "networking/net_admin_cap_pod.yaml"
+    When I run oc create as admin over "net_admin_cap_pod.yaml" replacing paths:
+       | ["spec"]["nodeName"]                                       | <%= cb.workers[1].name %> |
+       | ["metadata"]["namespace"]                                  | <%= project.name %>       |
+       | ["metadata"]["name"]                                       | hostnw-pod-worker1        |
+       | ["spec"]["containers"][0]["securityContext"]["privileged"] | true                      |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+       | name=network-pod |
+    And evaluation of `pod.name` is stored in the :hostnw_pod_worker1 clipboard
+    #capturing tcpdump for 2 seconds
+    Given I wait up to 60 seconds for the steps to pass:
+    """
+    When admin executes on the "<%= cb.hostnw_pod_worker1 %>" pod:
+       | bash | -c | timeout  --preserve-status 2 tcpdump -i <%= cb.default_interface %> esp |
+    Then the step should succeed 
+    # Example ESP packet un-encrypted will look like 16:37:16.309297 IP ip-10-0-x-x.us-east-2.compute.internal > ip-10-0-x-x.us-east-2.compute.internal: ESP(spi=0xf50c771c,seq=0xfaad)
+    And the output should contain: 
+       | <%= cb.workers[0].name %> > <%= cb.workers[1].name %>: ESP |
```

Comment:
Do we need regex, similar to @zhaozhanqi fix?
```ruby
    # Example ESP packet un-encrypted will look like 16:37:16.309297 IP ip-10-0-x-x.us-east-2.compute.internal > ip-10-0-x-x.us-east-2.compute.internal: ESP(spi=0xf50c771c,seq=0xfaad)
    And the output should match: 
       | <%= cb.workers[0].name %>.* > <%= cb.workers[1].name %>.*: ESP |
```

Reply:
Beside the new steps of enable/disable ipsec, all other steps are copied from https://github.com/openshift/verification-tests/blob/master/features/networking/ovn_ipsec.feature#L115, wait the response from @anuragthehatter

Reply:
@weliang1 the case you copied already be updated as ross mentioned.  guess you were copied the old version.

Reply:
@weliang1 please help update line 223, reference line 159

Reply:
Fixed in new code.